docs/localization/LOCALIZATION.md

```diff
@@ -737,12 +710,10 @@ fr-BE **fallbacks to** fr-FR, as fallback locales mapping unavailable, first nea
 
 bn-BD **fallbacks to** ar-AR, as it is the default fallback.
 
-
-## Scenario 2: Fallback based on **supportedLocales** language code.
+## Scenario 2: Fallback based on **supportedLocales** language code
 
 In case if **fallbackLocalesMap** is not provided and the targeted translation language is unavailable is supported locales list, The targeted translation language will be matched with supported locales **language** to find out the first nearest match, the **locale** can be different. If none match, it will fall back to the **default language**.
```

Comment:
```suggestion
If **fallbackLocalesMap** is not provided and the targeted translation language is not part of the supported locale list, the targeted translation language will be matched to the supported locales' **languages** to find out the first nearest one. Again, only the language code will be matched and the locale will be ignored. 

If none match, it will fallback to the **default language**.
```